tutorials/replication.md

```diff
@@ -236,8 +236,12 @@ in many cases stronger consistency between the primary and replicas is
 required. Under heavy workloads, replicas can lag far behind the primary,
 providing stale data to clients reading from the replicas. Moreover, in cases
 where any data loss is fatal, asynchronous replication may not provide enough
-of a durability guarantee. Luckily `synchronous_commit` has several options
-with varying consitency/performance tradeoffs:
+of a durability guarantee. Luckily [synchronous-commit][postgres-synchronous-commit-docs] has several options
```

Comment:
should be with underscore instead of dash?  Also, you can include the backticks inside the brackets to format the link text as code-style